### DISCUSSION TITLE
Documentation in a root project. Mistake or Success #16994

### DISCUSSION THREAD

**Discussion by jonathan-arenas-edo** [mkdocs/mkdocs#3162] [Q&A]
Hello people,

I need your help, before I found on last topics for no duplicate the same discussion.
I am implementing backsateg solution in my company and I am very happy with PoC result.
Now the next steps, I have a doubt about the documentation (Mkdocs)
Backstage use a Mkdocs for this solution, but I have a big problem, the documentation for all applications are in root project in a README.md file, but Mkdocs must assign the documentation in a folder, by default "docs".

What is the purpose of this configuration? What is the problem with assigning in root directory?
I read that it is a security issue, but I do not know where the security issue is.

Thank you very much for your answer and your time.
Kind regards.

**Comment by oprypin** [comment]
https://github.com/oprypin/mkdocs-same-dir

**Reply by jonathan-arenas-edo** [reply]
Ohhhh thank you so much @oprypin :)

I'm going to check it.

**Comment by jonathan-arenas-edo** [comment]
Hi! Thank you so much @oprypin , is worlking for me.

Now I have one doubt more, is it possible to create a folder config, for example "backstage" and inside create a mkdocs.yaml where this file read all md files on root project? 

Similar to this config: docs_dir: ../. , but is not working because is nor permit to start with ..

Do you know how I can to do it?
